search-app/src/cmr/search/api/routes.clj

```diff
@@ -365,6 +376,11 @@
                {params :params headers :headers ctx :request-context}
                (get-deleted-collections ctx path-w-extension params headers)))
 
+        (context ["/:path-w-extension" :path-w-extension #"(?:collections-created-after)(?:\..+)?"] [path-w-extension]
```

Comment:
We debated about this approach, but we wanted to be consistent with the approach you used for harvesting with "collections deleted after a given time". You used a separate endpoint for that instead of a parameter.